src/style.css

```diff
@@ -0,0 +1,114 @@
+@font-face {
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-family: "Avenir-Book";
+}
+
+@font-face {
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+  font-family: "Avenir-Heavy";
+}
+
+html {
+
+  font-family: "Avenir-Book", sans-serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.small {
+  margin-top: 20px;
+}
+
+.biginput {
+  color: #3d4e61;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  background: url(images/Search.svg) no-repeat scroll 7px 7px;
+  background-position-y: center;
+  background-position-x: 26px;
+  background-size: 19px 19px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  outline: none;
+}
+
+.img1 {
+  margin-left: 26px;
+}
+
+.biginput:hover {
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  background: url(images/Search.svg) no-repeat scroll 7px 7px;
+  background-position-y: center;
+  background-position-x: 26px;
+  font-size: 16px;
+  line-height: 22px;
+  outline: none;
+}
```

Comment:
do not duplicate properties to different state of the same element, only add that one, which changing